src/templater.rs

```diff
@@ -429,11 +429,11 @@ impl CommitOrChangeId {
 
 fn highlight_shortest_prefix(mut hex: String, total_len: usize, repo: RepoRef) -> String {
```

Comment:
It seems I did create a mess here. What I should have done is changed the `12` above to `13` together with your change here.

Feel free to change it to 13 or leave it at 12, whichever you prefer.

Reply:
or change the meaning of `total_len` to the number of hex characters excluding brackets?
I think that's simpler and more consistent with the default "short" hex length.

Reply:
I think I care more about the amount of space the ids take up than the number of letters in them (as long as our repos are not humongous). So, I'd rather maintain the sort of consistency where they take up the same amount of space. Taking up 4 extra characters per line for the two ids means more lines wrap on a narrow-ish screen. I wasn't completely comfortable when I decided to take up 2 extra characters per line.

I don't know how much I care. Most people will hopefully end up using 12 characters with colors. It could even be configurable how wide the ids are -- which could be nice for a per-repo config.

Reply:
Ok, let's leave it as is (`total_len = 12`) then. My concern was just what this `- 1` means.

Hopefully this will be replaced with prefix highlighting, and we won't have to consider this kind of things again.